Image upload in Euth is broken

BE Error 400: "Claimed MIME type is image/jpeg but file content seems to be application/octet-stream"

I cannot test it locally, build error: src/euth/euth/build/js/AdhocracySpec.ts(19,22): error TS2307: Cannot find module './Packages/LocalSocket/LocalSocketSpec'.
The acceptance test on master were passing yesterday,   maybe this is a deployment issue, is https://github.com/ahupp/python-magic isntalled and working properly (@slomo) ?

--- a/docs/development/installation.rst
+++ b/docs/development/installation.rst
@@ -12,11 +12,12 @@ Requirements (Tested on Debian\Ubuntu,  64-Bit is mandatory):
 4. graphviz
 5. ruby ruby-dev
 6. gettext
+7. libmagic1
 
 If you don't use the custom compiled python (see below) you need
 some basic dependencies to build PIL (python image library):
 
-6. libjpeg8-dev zlib1g-dev (http://pillow.readthedocs.org/en/latest/installation.html)
+8. libjpeg8-dev zlib1g-dev (http://pillow.readthedocs.org/en/latest/installation.html)
 
 Create SSH key and upload to GitHub ::
 

